TLM/TLM/Resources/lang_pl.txt

```diff
@@ -233,10 +233,24 @@ Ignore_disabled_mods Ignoruj wyłączone mody
 Traffic_Manager_detected_incompatible_mods wykrył niekompatybilne mody
 Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
 Unsubscribe Odsubskrybuj
+Keybind_toggle_TMPE_main_menu Menu TM:PE
+Keybind_toggle_traffic_lights_tool Narzędzie 'Przełącznik Sygnalizacji świetlnej'
+Keybind_use_lane_arrow_tool Narzędzie 'Strzałki pasów ruchu'
+Keybind_use_lane_connections_tool Narzędzie 'Połącz pasy ruchu'
+Keybind_use_priority_signs_tool Narzędzie 'Znaki pierwszeństwa'
+Keybind_use_junction_restrictions_tool Narzędzie 'Ograniczenia na skrzyżowaniu'
+Keybind_use_speed_limits_tool Narzędzie 'Limity prędkości'
+Keybind_lane_connector_stay_in_lane Połącz pasy ruchu: Pozostań na pasie
+Keybind_lane_connector_delete Oczyść połączenia pasów ruchu
+Keybinds Skróty klawiszowe
+Keybind_Exit_subtool Wyłącz narzędzie i zamknij TM:PE menu
+Keybind_conflict The key you've selected is conflicting with another shortcut key. Please choose something else.
```

Comment:
```suggestion
Keybind_conflict Wybrany skrót klawiszowy jest już używany. Wybierz inną kombinację.
```

Reply:
Added

Reply:
> I noticed in code some stuff about row striping, but didn't see any on the UI? (not important, just mentioning it in case the rows were supposed to be striped)

The row striping code is still there. But in the old dialog the default background was grey-ish, and row striping code just cleared texture on some lines. In the new dialog i've no idea what the background sprite name is, so they are all black (i.e. not striped). Have to go into ModTool or UIHelper source and find the background sprite name which is used in other dialogs.